helm/athena/templates/ingress.yaml

```diff
@@ -1,4 +1,8 @@
+{{- if .Capabilities.APIVersions.Has "networking.k8s.io/v1/Ingress" }}
```

Comment:
Do we actually need all these conditionals if we only want to run this on 1.19 and higher?

Reply:
Hmm. I based my comment on "This makes the chart incompatible with Kubernetes < 1.19" ind the PR description and a changelog entry, which was outdated it seems.

Now I am confused whether or not the chart is meant to run on Kubernetes older than 1.19.

Reply:
The Ingress apiVersion used in the latest version of athena is compatible with Kubernetes older than 1.19. This change makes it possible to install athena on clusters above 1.22 (See https://kubernetes.io/docs/reference/using-api/deprecation-guide/#ingress-v122)

Thanks for noticing and raising!